org.opentosca.planbuilder.core.bpel/src/org/opentosca/planbuilder/core/bpel/BPELBuildProcessBuilder.java

```diff
@@ -117,8 +117,9 @@ public int registeredPlugins() {
     public BPELPlan buildPlan(final String csarName, final AbstractDefinitions definitions,
                               final QName serviceTemplateId) {
         // create empty plan from servicetemplate and add definitions
-
+    	BPELBuildProcessBuilder.LOG.debug("Building plan for {} with serviveTemplateId {}", csarName, serviceTemplateId.getNamespaceURI());
```

Comment:
indentation is not according to formatting conventions

Reply:
the Logger is also overqualified.

I'd prefer to indent like the surrounding code and to not overqualify the Logger call.
Generally that is. I won't explicitly note occurrences of this

Reply:
Actually this commit is not part of the current pull request. Seems like I messed something up when creating the branch. Sorry for that!